concepts/validator/index.md

```diff
@@ -233,7 +233,14 @@ Autonity implements an 'active epoch' staking model, applying staking transition
+Stake is bonded by submitting a bonding request transaction to the [`bond()`](/reference/api/aut/#bond) function and redeemed by the converse, a transaction to the [`unbond()`](/reference/api/aut/#unbond) function. Bonding and unbonding is an incremental process.
+
+In the case of bonding: newton for the stake [delegation](/glossary/#delegation) amount is burned immediately on processing of the `bond()` function call and the corresponding amount of Newton is deducted from the stake delegator's Newton account balance. If the bonding request is included in block `T`, then actual bonding is executed at epoch end, i.e. `T` + remainder of the epoch. The validator's total bonded stake amount is then updated at epoch end when the staking transition is applied. When bonding liquid newton is minted for [delegated](/glossary/#delegated) stake only. The liquid newton is minted to the stake delegator's account at epoch end per the NTN-LNTN conversion rate for the validator (see [Staking, Liquid Newton](/concepts/staking/#liquid-newton)). Liquid newton is not minted for [self-bonded](/glossary/#self-bonded) stake.
+
+In the case of unbonding: if the unbonding request is included in block `T`, then actual unbonding is then executed at `T` + [`unbondingPeriod`](/concepts/staking/#unbondingperiod) + remainder of the [`epoch`](/concepts/staking/#epoch) in which the unbonding period falls. At this point, the staker's due newton is redeemed and minted to them. The redemption flow varies depending on whether the unbonding is for [self-bonded](/glossary/#self-bonded) or [delegated](/glossary/#delegated) stake:
+
+- For [self-bonded](/glossary/#self-bonded) stake, the unbonding request amount is stated in NTN and that amount of newton is redeemed at epoch end.
+- For [delegated](/glossary/#delegated) stake, the unbonding request amount is stated in LNTN and the amount of newton redeemed at epoch end depends on the NTN-LNTN conversion rate for the validator (see [Staking, Liquid Newton](/concepts/staking/#liquid-newton)). Liquid newton for [delegated](/glossary/#delegated) stake is burned immediately on processing of the [`unbond()`](/reference/api/aut/#unbond) function call. The validator's total bonded stake amount is then updated at `T` + [`unbondingPeriod`](/concepts/staking/#unbondingperiod) + remainder of the [`epoch`](/concepts/staking/#epoch) in which the unbonding period falls. At this point, the staker's due newton is redeemed and minted to them.
```

Comment:
> Liquid newton for [delegated](/glossary/#delegated) stake is burned immediately on processing of the [`unbond()`](/reference/api/aut/#unbond) function call.

Not really, it is burned at epoch end after the `unbond()`

Reply:
> The validator's total bonded stake amount is then updated at `T` + [`unbondingPeriod`](/concepts/staking/#unbondingperiod) + remainder of the [`epoch`](/concepts/staking/#epoch) in which the unbonding period falls.

This kind of gives the idea that the unbonding stake is still accounted for voting power computation, but it is not the case. as soon as the epoch end's after the `unbond()` tx is mined, the bonded stake is reduced.

Reply:
Yes. Reworded to:

>- For [delegated](/glossary/#delegated) stake, the unbonding request amount is stated in LNTN and the amount of newton redeemed at epoch end depends on the NTN-LNTN conversion rate for the validator (see [Staking, Liquid Newton](/concepts/staking/#liquid-newton)). At the end of the [`epoch`](/concepts/staking/#epoch) in which the [`unbond()`](/reference/api/aut/#unbond) transaction was processed the Liquid Newton is burned and the validator's total bonded stake amount is reduced. At `T` + [`unbondingPeriod`](/concepts/staking/#unbondingperiod) + remainder of the [`epoch`](/concepts/staking/#epoch) in which the unbonding period falls, the staker's due newton is redeemed and minted to them.

Reply:
Nope, rephrased this section to:

>- For [delegated](/glossary/#delegated) stake, the unbonding request amount is stated in LNTN and the amount of newton redeemed depends upon the NTN-LNTN conversion rate for the validator (see [Staking, Liquid Newton](/concepts/staking/#liquid-newton)). At the end of the [`epoch`](/concepts/staking/#epoch) in which the [`unbond()`](/reference/api/aut/#unbond) transaction was processed the Liquid Newton is burned and the validator's total bonded stake amount is reduced. The due amount of newton is redeemed at epoch end after unbonding period expiry.

Reply:
Done in commit e023406f88dc4d2c9188ed9616cecd8be614327a.